tests/e2e/lib/pages/wpcom/login.js

```diff
@@ -28,34 +22,23 @@ export default class LoginPage extends Page {
 
 		const usernameSelector = '#usernameOrEmail';
 		const passwordSelector = '#password';
-		const continueButtonSelector = '.login__form-action button';
-		const submitButtonSelector = '.login__form-action button[type="submit"]';
+		const continueButtonSelector = '//button[text()="Continue"]';
```

Comment:
Why these locators were updated? 

Do you think using button text is reliable?

Reply:
Both continue and submit buttons have the same class `.login__form-action button`, both of type submit. I wanted clear different locators for them. I don't think the text of these 2 buttons should change too often. So I find it acceptable.

Reply:
yeah, makes sense 👌